gluten-core/src/main/scala/org/apache/spark/util/TaskResources.scala

```diff
@@ -164,44 +162,39 @@ object TaskResources extends TaskListener with Logging {
 // thread safe
 class TaskResourceRegistry extends Logging {
   private val sharedUsage = new SimpleMemoryUsageRecorder()
-  private val resources = new java.util.LinkedHashMap[String, TaskResource]()
```

Comment:
no requirements to use LinkedHashMap, so changed to HashMap

Reply:
👍 

It was from a legacy code version using linkedhashmap to implement fifo, if I remember correctly.